docusaurus/docs/deployment.md

```diff
@@ -155,6 +156,19 @@ You can specify other environments in the same way.
 
 Variables in `.env.production` will be used as fallback because `NODE_ENV` will always be set to `production` for a build.
 
+## [AWS Amplify](http://console.amplify.aws)
+
+The AWS Amplify Console provides continuous deployment and hosting for modern web apps (single page apps and static site generators) with serverless backends. The Amplify Console offers globally available CDNs, easy custom domain setup, feature branch deployments, and password protection.
+
+1. Login to the Amplify Console [here](https://console.aws.amazon.com/amplify/home).
+1. Connect your create-react-app repo and pick a branch. If you're looking for a CRA+Amplify starter, try the [create-react-app-auth-amplify starter](https://github.com/swaminator/create-react-app-auth-amplify) that demonstrates setting up auth in 10 minutes with create-react-app.
+1. The Amplify Console automatically detects the build settings. Choose Next.
+1. Choose *Save and deploy* and sit back.
+
+If the build succeeds, the app is deployed and hosted on a global CDN with an amplifyapp.com domain. You can now continuously deploy changes to your frontend or backend. Continuous deployment allows developers to deploy updates to their frontend and backend on every code commit to their Git repository. 
+
+<img src="https://raw.githubusercontent.com/swaminator/create-react-app-auth-amplify/master/src/images/amplifyconsole-cra.gif" width="500" alt="amplify-console">

```

Comment:
I don't think this GIF is adding much. It moves way to fast to really be able to tell what's going on. Please remove it.